runtime-modules/working-group/src/lib.rs

```diff
@@ -542,6 +571,17 @@ decl_module! {
             // == MUTATION SAFE ==
             //
 
+            if opening.opening_type == OpeningType::Regular {
+                // Lead must be set for ensure_origin_for_openig_type in the
+                // case of regular.
+                let lead = Self::worker_by_id(Self::current_lead().unwrap());
+                let current_stake = T::StakingHandler::current_stake(&lead.staking_account_id);
+                T::StakingHandler::set_stake(
+                    &lead.staking_account_id,
+                    current_stake.saturating_sub(T::OpeningStake::get())
```

Comment:
What if the `OpeningStake` constant changes between add_opening() and fill_opening() because of the runtime upgrade?